src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/DefaultTagHelperDescriptionFactory.cs

```diff
@@ -164,11 +164,12 @@ internal static string CleanSummaryContent(string summaryContent)
             // if there's a <para> in the summary element when it's shown in the completion description window
             // it'll be serialized as html (wont show).
 
-            var crefMatches = ExtractCrefRegex.Value.Matches(summaryContent).Reverse();
+            var crefMatches = ExtractCrefRegex.Value.Matches(summaryContent);
             var summaryBuilder = new StringBuilder(summaryContent);
 
-            foreach (var cref in crefMatches)
+            for (var i = crefMatches.Count - 1; i >= 0; i--)
```

Comment:
Why do it this way instead of via Reverse? I'm sure it's more performant, but it's a bit harder to tell what's going on. Maybe a comment explaining why it needs to be reversed and why we did it this way?

Reply:
Reverse isn't available in netstandard2.0 so we have to manually reverse like this. That being said this is a superior way to reversing anyways because we don't allocate an enumerator for the reverse.